torch/_inductor/select_algorithm.py

```diff
@@ -1901,6 +1903,19 @@ def bind(
             self, input_nodes, layout, kwargs, has_out_variant=self.has_out_variant
         )
 
+    def maybe_append_choice(
```

Comment:
Is this used or tested anywhere?  PRs should include usage/tests and not add unused code.